tunnel/dns/server.go

```diff
@@ -36,7 +37,23 @@ type resolver struct {
 	namesMtx sync.Mutex
 }
 
+func flushDnsCaches() {
+	bin, err := exec.LookPath("systemd-resolve")
```

Comment:
does this 'force' us to use debian (ubuntu?) images?  not a problem for us or users using our appliance but for customers "doing it on their own" they should at least know this but i don't know how to communicate that to the end-user effectively... :/

Reply:
It won't break anything, it'll just spit out an error message if can't run it. Should I enhance the error message to say something like: `'error flushing dns, consider adding a dns-flush to your restart script'`?